packages/svelte/tests/runtime-runes/samples/proxy-prop-default-readonly/_config.js

```diff
@@ -0,0 +1,18 @@
+import { test } from '../../test';
+
+export default test({
+	html: `<button>clicks: 0</button>`,
+
+	compileOptions: {
+		dev: true
+	},
+
+	async test({ assert, target }) {
+		const btn = target.querySelector('button');
+		await btn?.click();
+
+		assert.htmlEqual(target.innerHTML, `<button>clicks: 0</button>`);
+	},
+
+	runtime_error: 'Props are read-only, unless used with `bind:`'
```

Comment:
How do we bring the nuance across in this error message that reassignment is ok? Also, the test should probably have another button that tests that this is ok

Reply:
How about this?

```suggestion
	runtime_error: 'Props cannot be mutated, unless used with `bind:`'
```

Reply:
Reads well to me.